t/api/11-bugs.t

```diff
@@ -71,4 +71,8 @@ $t->get_ok('/api/v1/bugs/2')->status_is(404, 'Bug #2 deleted');
 
 $t->delete_ok('/api/v1/bugs/2')->status_is(404, 'Bug #2 already deleted');
 
+$t->post_ok('/api/v1/jobs/99926/comments', form => {text => 'Böser Fehler: jsc#SLE-42999'});
```

Comment:
Perhaps avoid the use of German here which potentially confuses people? `Really bad bug` conveys the same point I think.

Reply:
Agreed, not using english in a project with so many contributors is a bit rude.

Reply:
(unless you verify foreign language support, which you don't)

Reply:
@kalikiana I go for 'wicked bug' then. That translation is closer and it creates a nice ambiguity with our network management tool.